<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>header TO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="form_headerto">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>170</y>
      <width>146</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>173</y>
      <width>93</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>header FROM</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="form_headerfrom">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>146</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="form_mailserver_port">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>24</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>24</y>
      <width>88</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>* Mail Server</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="form_mailserver">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>24</y>
      <width>146</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>26</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>448</y>
      <width>63</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>* Subject</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="form_ehlo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>57</y>
      <width>146</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="form_start">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>723</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="form_replyto">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>146</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>393</x>
      <y>690</y>
      <width>30</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>mps</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="form_subject">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>448</y>
      <width>421</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="mps">
    <property name="geometry">
     <rect>
      <x>429</x>
      <y>690</y>
      <width>107</width>
      <height>27</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>420000000</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>* RCPT TO</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="form_data">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>481</y>
      <width>421</width>
      <height>192</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="form_mailfrom">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>107</y>
      <width>146</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>690</y>
      <width>36</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Total</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>57</y>
      <width>38</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>EHLO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="form_rcptto">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>146</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="form_stop">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>723</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>93</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>* MAIL FROM</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="form_xheader">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>421</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Reply-to</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="form_returnpath">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>210</y>
      <width>146</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="mail_total">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>690</y>
      <width>107</width>
      <height>27</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>420000000</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>481</y>
      <width>75</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>* Contents</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>65</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>X-Header</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>83</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Return-path</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>541</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>541</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>541</width>
      <height>521</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>frame_3</zorder>
   <zorder>frame</zorder>
   <zorder>label_11</zorder>
   <zorder>form_headerto</zorder>
   <zorder>label_12</zorder>
   <zorder>form_headerfrom</zorder>
   <zorder>form_mailserver_port</zorder>
   <zorder>label_6</zorder>
   <zorder>form_mailserver</zorder>
   <zorder>label_13</zorder>
   <zorder>label_7</zorder>
   <zorder>form_ehlo</zorder>
   <zorder>form_start</zorder>
   <zorder>form_replyto</zorder>
   <zorder>label_9</zorder>
   <zorder>form_subject</zorder>
   <zorder>mps</zorder>
   <zorder>label_8</zorder>
   <zorder>form_data</zorder>
   <zorder>form_mailfrom</zorder>
   <zorder>label_3</zorder>
   <zorder>label</zorder>
   <zorder>form_rcptto</zorder>
   <zorder>form_stop</zorder>
   <zorder>label_5</zorder>
   <zorder>form_xheader</zorder>
   <zorder>label_14</zorder>
   <zorder>form_returnpath</zorder>
   <zorder>mail_total</zorder>
   <zorder>label_4</zorder>
   <zorder>label_16</zorder>
   <zorder>label_15</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>582</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMail_Attack_v0_1">
    <property name="title">
     <string>help</string>
    </property>
    <addaction name="actionInformation"/>
    <addaction name="actionLicense"/>
   </widget>
   <addaction name="menuMail_Attack_v0_1"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionInformation">
   <property name="text">
    <string>help</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>license</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>form_mailserver</tabstop>
  <tabstop>form_mailserver_port</tabstop>
  <tabstop>form_ehlo</tabstop>
  <tabstop>form_mailfrom</tabstop>
  <tabstop>form_rcptto</tabstop>
  <tabstop>form_headerfrom</tabstop>
  <tabstop>form_headerto</tabstop>
  <tabstop>form_replyto</tabstop>
  <tabstop>form_returnpath</tabstop>
  <tabstop>form_xheader</tabstop>
  <tabstop>form_subject</tabstop>
  <tabstop>form_data</tabstop>
  <tabstop>mail_total</tabstop>
  <tabstop>mps</tabstop>
  <tabstop>form_stop</tabstop>
  <tabstop>form_start</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
